src/dmd/root/filename.d

```diff
@@ -653,6 +657,38 @@ nothrow:
         return null;
     }
 
+    extern (D) static const(char)[] searchPath(const(char)* path, const(char)[] name, bool cwd)
```

Comment:
Can the array version call this version?

Reply:
It's a good idea, I was thinking about evolving it in that direction.